_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,89 @@
+## Android setup
+
+_This step is optional, and can be performed at any time before
+using Flutter to develop apps for Android._
+
+### Install Android Studio
+
+To develop Flutter apps for Android, you need to install Android Studio. After installation,
+you also need to start Android Studio once, to 
+
+1. Install [Android Studio](https://developer.android.com/studio/index.html).
+
+   Flutter requires the latest Android SDK and Android SDK Platform-Tools packages.
+   You can verify these are up-to-date in **Android Studio>Configure>SDK Manager**,
+   as described in [Update the IDE and SDK Tools](https://developer.android.com/studio/intro/update.html).
+
+1. Start Android Studio.
+
+### Set up the Android emulator
+
+To prepare to run and test your Flutter app on the Android emulator, follow these steps:
+1. Enable Kernel-based Virtual Machine (KVM) support on your machine.
+1. Define a virtual device in Android Virtual Device Manager.
+
+#### Enable KVM
+For the Android emulator, you need to configure x86 and x86_64 emulation, which 
+requires the use of hardware virtualization. The steps to enable hardware virtualization 
+are:
+
+1. Reboot into the system BIOS (usually by repeatedly tapping F1 or F10 at the first page during startup).
+1. In **Security > System Security**:
```

Comment:
Will note that... this particular section is preliminary, since it's high-impact to test, and most Flutter users are not on Linux. So it's for the small subset of users who are on Linux, don't have KVM enabled already, and prefer using an emulator vs. a device.